openapi/ynab/types.bal

```diff
@@ -0,0 +1,604 @@
+public type PayeeLocation record {
```

Comment:
license header missing

Reply:
Resolved & amended the changes. Thank you.